index.js

```diff
@@ -93,8 +93,8 @@ const destroyCircular = ({
 	});
 
 	for (const [key, value] of Object.entries(from)) {
-		// eslint-disable-next-line node/prefer-global/buffer
-		if (typeof Buffer === 'function' && Buffer.isBuffer(value)) {
+		if (value !== null && value.constructor !== null &&
```

Comment:
I would go with `value && value instanceof Uint8Array && value.constructor.name === 'Buffer'`

Reply:
I updated the logic as per your recommendation